RFCs/0024-Blame-Collector-Options.md

```diff
@@ -0,0 +1,71 @@
+# 0024 Blame collector options
+
+## Summary
+This note outlines the proposed changes for adding more options to test blame data collector.
```

Comment:
`changes for adding more options to test blame data collector. `> `changes for enhancing blame data collector options.`